lib/aws/bootstrap/chart_values/external-dns.j2.yaml

```diff
@@ -1,19 +1,45 @@
+## @section Global parameters
```

Comment:
please only set what's really useful in order to reduce the size and make it more visible. thx